_clients/javascript/helpers.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: default
+title: Helpers
+parent: JavaScript client
+nav_order: 2
+---
+
+# Helpers
+
+This section is currently incomplete. It does not yet cover all helper methods and is missing documentation of various helper options.
+{: .warning }
+
+## Bulk helper
+
+Running bulk requests can be complex due to the shape of the API, this helper aims to provide a nicer developer experience around the Bulk API.
```

Comment:
Maybe simpler to say _The `bulk` helper simplifies making complex bulk API requests._?